packages/blue-sdk-ethers-liquidation/examples/whitelisted-erc4626-1inch.ts

```diff
@@ -151,10 +155,80 @@ export const check = async (
           triedLiquidity.map(
             async ({ seizedAssets, repaidAssets, withdrawnAssets }) => {
               try {
-                const srcToken =
+                let srcToken =
                   collateralUnderlyingAsset ?? market.config.collateralToken;
-                const srcAmount = withdrawnAssets ?? seizedAssets;
+                let srcAmount = withdrawnAssets ?? seizedAssets;
+
+                const encoder = new LiquidationEncoder(executorAddress, signer);
 
+                let dstAmount = 0n;
+                // Handle Pendle Tokens
+                // To retrieve the tokens, we need to call the Pendle API to get the swap calldata
+                if (pendleTokens[chainId].has(market.config.collateralToken)) {
+                  const pendleMarketData =
+                    pendleMarkets[chainId][market.config.collateralToken];
+                  const maturity = pendleMarketData?.maturity;
+                  if (!maturity) {
+                    throw Error("Pendle market not found");
+                  }
```

Comment:
We know it's a pendle token but we could not find the market: it throws.
We might want to fall back one the swap aggregator who maybe has a better chance at swapping than us?

Reply:
Will add to the future version of the bot